test-app/presenter/src/commonMain/kotlin/com/example/redwood/testing/presenter/TestApp.kt

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (C) 2023 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.example.redwood.testing.presenter
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.MutableState
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import app.cash.redwood.Modifier
+import app.cash.redwood.layout.api.Constraint
+import app.cash.redwood.layout.api.Constraint.Companion
+import app.cash.redwood.layout.api.CrossAxisAlignment
+import app.cash.redwood.layout.api.Overflow
+import app.cash.redwood.layout.compose.Column
+import app.cash.redwood.ui.Margin
+import app.cash.redwood.ui.dp
+import com.example.redwood.testing.compose.Button
+import com.example.redwood.testing.compose.Text
+
+@Composable
+fun TestApp(httpClient: HttpClient) {
+  val screen = remember { mutableStateOf<Screen?>(Screen.RepoSearch) }
```

Comment:
Should we default to showing the `HomeScreen`?

```suggestion
  val screen = remember { mutableStateOf<Screen?>(null) }
```

Reply:
Yes. Was testing if this fixed iOS. It didn't.